requirements.txt

```diff
@@ -17,3 +17,5 @@ sklearn
 sympy
 tensorflow
 xgboost
```

Comment:
Please insert requirements in alphabetical order to reduce the likelihood of duplicate entries.